pkgs/development/python-modules/pytest-watch/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, pkgs
+, pythonPackages
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-watch";
+  version = "4.2.0";
+  disabled = pythonOlder "3.5";
```

Comment:
I think this supports python 2.7 https://github.com/joeyespo/pytest-watch/pull/95

Reply:
The snag is that `coverage` for Python 2.7 seems not to be the right version (<5, >=4) that `pytest-testmon` requires.
```
/build/pytest-testmon-0.9.16/dist /build/pytest-testmon-0.9.16
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
Processing ./pytest_testmon-0.9.16-py2-none-any.whl
Collecting coverage<5,>=4 (from pytest-testmon==0.9.16)
  Could not find a version that satisfies the requirement coverage<5,>=4 (from pytest-testmon==0.9.16) (from versions: )
No matching distribution found for coverage<5,>=4 (from pytest-testmon==0.9.16)

```